### ISSUE TITLE
Can't connect to virtual USB serial port

### ISSUE THREAD

**Issue by fhunleth** [nerves-project/nerves_system_rpi0#10]
When the Raspberry Pi Zero is connected to my Linux laptop, I see errors like this in `dmesg`:

```
[  463.159931] usb 2-1.7: new high-speed USB device number 5 using ehci-pci
[  468.231677] usb 2-1.7: device descriptor read/64, error -110
[  483.406923] usb 2-1.7: device descriptor read/64, error -110
```
On the Pi Zero, it looks like it's trying to enumerate the laptop:
```
[   22.412560] dwc2 20980000.usb: new device is high-speed
[   22.420708] dwc2 20980000.usb: dwc2_hsotg_enqueue_setup: failed queue (-11)
[   22.588402] dwc2 20980000.usb: new device is high-speed
[   22.596563] dwc2 20980000.usb: dwc2_hsotg_enqueue_setup: failed queue (-11)
[   27.660725] dwc2 20980000.usb: new device is high-speed
[   27.668904] dwc2 20980000.usb: dwc2_hsotg_enqueue_setup: failed queue (-11)
[   42.836557] dwc2 20980000.usb: new device is high-speed
[   42.844617] dwc2 20980000.usb: dwc2_hsotg_enqueue_setup: failed queue (-11)
[   43.012399] dwc2 20980000.usb: new device is high-speed
[   43.020361] dwc2 20980000.usb: dwc2_hsotg_enqueue_setup: failed queue (-11)
[   53.492406] dwc2 20980000.usb: new device is high-speed
[   53.500478] dwc2 20980000.usb: dwc2_hsotg_enqueue_setup: failed queue (-11)
```

**Comment by fhunleth** [comment]
I did some more investigation. I can reproduce this on a Dell Precision M4700 laptop and an HP Omen. It works fine for me on an iMac. I also captured traces of the USB bus for working and non-working cases, and it looks like there's corruption in the dual-mode case that the iMac can get past but the other hardware can't. I'm including the traces below. Use the Total Phase Data Center software to view.

[usb-traces.zip](https://github.com/nerves-project/nerves_system_rpi0/files/1068267/usb-traces.zip)

The only glimmer of hope is that Raspbian somehow is able to make dual-mode work on all system, I think. It doesn't look like any special dwc2 module parameters are involved with Raspbian and experimenting with some kernel options didn't seem to do anything, though.
